### PR TITLE
Projects Section Part

### PR DESCRIPTION
**Pull request by ksdkamesh99**
I have completed the Projects Section.

I hereby attach a screenshot.
image
Kindly review it and merge

### CODE REVIEW

File: _includes/featured_projects.html [hunk 1]
```diff
@@ -1,3 +1,48 @@
-<div class="featured_projects">
 
+<!-- HTML Setup -->
+<div class="featured_projects">			
+					<h2 class="text-center text-white mt-3">MegaMinds at Work</h2>										
+		
+	
+	<!-- cards -->
```
Comment (sachsom95) [inline _includes/featured_projects.html]: The file needs to be passed through an HTML linter for consistent spaces. Makes it easier to work with later

File: _sass/members.scss [hunk 2]
```diff
@@ -1,9 +1,4 @@
-* {
-	margin: 0;
-	padding: 0;
-	-webkit-box-sizing: border-box;
-	box-sizing: border-box;
-}
+
 h1 {
 	font-size: 2.5rem;
```
Comment (sachsom95) [inline _sass/members.scss]: H1 tag seems to have a global css value is it intentional?

File: _sass/projects.scss [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+@import 'variables';
+// Variables
+$font-family-base : 'Raleway', sans-serif;
+$font-family-head-base : 'Nothing You Could Do', cursive; // Custom
+$font-family-lead-base : 'Montserrat', sans-serif; // Custom
+// colors
+$white:    #fff !default;
+$warning : #ffc107;
+// Font Weight Base
+$font-weight-light:           100 !default;
```
Comment (sachsom95) [inline _sass/projects.scss]: is !default required?

Comment (ksdkamesh99) [reply _sass/projects.scss]: Yes

Comment (sachsom95) [reply _sass/projects.scss]: okay I will resolve the this

File: _sass/members.scss (hunk 2 above)
Comment (ksdkamesh99) [reply _sass/members.scss]: Actually it is the code of suhani

Comment (sachsom95) [reply _sass/members.scss]: ohh I see @suhanichawla will u be changing it?

Comment (suhanichawla) [reply _sass/members.scss]: I can change it at the end, just need to preceed it with .wrapper class

**Pull request merged**